src/main/java/florizz/objects/Flower.java

```diff
@@ -6,6 +6,7 @@ public class Flower {
     private String name = "";
     private ArrayList<Occasion> occasions = null;
     private Colour colour = null;
+    private String price = "";
```

Comment:
May want to store price as a double? to better facilitate later logic like total cost of bouquet

Reply:
Good idea